yarn-project/ethereum/src/l1_tx_utils.ts

```diff
@@ -209,6 +213,8 @@ export class L1TxUtils {
 
       if (gasConfig.fixedGas) {
         gasLimit = gasConfig.fixedGas;
+      } else if (gasConfig.gasLimit) {
+        gasLimit = this.bumpGasLimit(gasConfig.gasLimit, gasConfig);
```

Comment:
```suggestion
      } else if (gasConfig.gasLimit) {
        gasLimit = gasConfig.gasLimit;
```

I find it confusing that if I set a fixed value, the library then bumps it. I'd expect it to use the exact fixed value I requested. Open to discussion though!

Reply:
I was on the fence on this, I kept `fixedGas` for this purpose but I might just be making it confusing.
The reason was, I wanted to bump the result of `simulateGasUsed` inside l1-publisher without having a fixed number.
Probably can just extract the gas settings inside l1-publisher though, and multiply it there